src/heur.py

```diff
@@ -47,7 +58,7 @@ def evaluate(self, x):
             self.best_x = x
         if y <= self.of.get_fstar():
             raise StopCriterion('Found solution with desired fstar value')
-        if self.neval == self.maxeval:
+        if self.neval >= self.maxeval:
```

Comment:
Have you found any case when the equality condition would not be enough?

(I know it is safer this way, I am just curious).